app/main/views/services.py

```diff
@@ -135,8 +135,13 @@ def update_service(service_id):
         service, update, update_details
     )
 
+    audit_data = {
+        'supplierName': service.supplier.name,
+        'supplierId': service.supplier.supplier_id
```

Comment:
`commit_and_archive_service` adds service id key `service_id`. We should probably change either that one or `supplierId` to have the same format.